code/modules/awaymissions/mission_code/dsbssigma.dm

```diff
@@ -0,0 +1,773 @@
+
+/obj/machinery/door/poddoor/shutters/rusted/do_animate(animation)
+	switch(animation)
+		if("opening")
+			flick("rusted_opening", src)
+			playsound(src, 'sound/machines/blastdoor.ogg', 30, 1)
+		if("closing")
+			flick("rusted_closing", src)
+			playsound(src, 'sound/machines/blastdoor.ogg', 30, 1)
+
+/obj/machinery/door/poddoor/shutters/rusted/update_icon()
+	if(density)
+		icon_state = "rusted_closed"
+	else
+		icon_state = "rusted_open"
+
+//////// MECHA ////////
+
+/obj/mecha/combat/gygax/dark/rusted
+	desc = "This once lightweight exosuit, painted in a dark scheme, has not been spared by the elements. Exterior armor is heavily rusted and most equipment sockets have decayed beyond repair."
+	name = "Rusted Dark Gygax"
+	max_integrity = 200
+	deflect_chance = 10
+	armor = list(melee = 20, bullet = 20, laser = 25, energy = 20, bomb = 10, bio = 0, rad = 10, fire = 50, acid = 50)
+	max_temperature = 25000
+	leg_overload_coeff = 50
+	operation_req_access = list(ACCESS_AWAY01)
+	internal_damage_threshold = 25
+	max_equip = 0
+	destruction_sleep_duration = 2
+
+/obj/mecha/combat/gygax/dark/rusted/add_cell()
+	cell = new /obj/item/stock_parts/cell/high(src)
+
+/obj/mecha/combat/gygax/dark/rusted/onTransitZ()
+	do_sparks(3, 1, src)
+	take_damage(300, BRUTE)
+
+////// STRUCTURES //////
+
+/obj/structure/displaycase/telecrystal
+	name = "officer's display case"
+	desc = "A display case containing a single telecrystal. A crimson reminder that no matter how big or small an atrocity is - someone, somewhere will sponsor it."
+	start_showpiece_type = /obj/item/stack/telecrystal
+	req_access = list(ACCESS_AWAY03)
+
+/obj/structure/closet/fireaxecabinet/empty
+	fireaxe = null
+	icon_closed = "fireaxe0000"
+	icon_opened = "fireaxe0140"
+	icon_state = "fireaxe0140"
+	locked = 0
+	opened = 1
+	localopened = TRUE
+
+/obj/structure/utility
+	name = "utility catwalk"
+	desc = "A utility catwalk for easier life support maintenance and cable access."
+	icon = 'icons/obj/smooth_structures/catwalk.dmi'
+	icon_state = "catwalk"
+	density = FALSE
+	anchored = TRUE
+	armor = list("melee" = 50, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 80, "acid" = 50)
+	max_integrity = 50
+	layer = 2.5
+	plane = -1
+	smooth = SMOOTH_TRUE
+	canSmoothWith = list(/obj/structure/utility,
+						/obj/structure/utility/broken/smooth)
+
+/obj/structure/utility/broken
+	desc = "A broken utility catwalk."
+	icon = 'icons/obj/smooth_structures/catwalk.dmi'
+	icon_state = "catwalk-b"
+	smooth = SMOOTH_FALSE
+	canSmoothWith = null
+
+/obj/structure/utility/broken/smooth
+	icon_state = "catwalk-bs"
+
+/obj/structure/utility/lattice
+	name = "collapsed ceiling"
+	desc = "A collapsed section of lightweight support lattice."
+	icon = 'icons/obj/smooth_structures/lattice.dmi'
+	icon_state = "lattice"
+	layer = 3.1
+	level = 2
+	smooth = SMOOTH_FALSE
+	canSmoothWith = null
+
+//////// LOOT ////////
+
+/obj/effect/spawner/lootdrop/away/dsbssigma/cleaning
```

Comment:
I feel it would be best to make a `/obj/effect/spawner/lootdrop/away/dsbssigma` base type and have it set the `lootcount` and `lootdoubles` variable so you don't have to repeat it for the others